os/net/netmgr/bsd_socket_api.c

```diff
@@ -49,84 +53,153 @@ int _create_netlink(int type, int protocol)
 	return fd;
 }
 
+sock_type _get_socktype(int fd)
+{
+	if (fd < CONFIG_NFILE_DESCRIPTORS) {
+		return LWNL;
+	} else if (fd < CONFIG_NUDS_DESCRIPTORS) {
```

Comment:
```suggestion
	} else if (fd < CONFIG_NFILE_DESCRIPTOR + CONFIG_NUDS_DESCRIPTORS) {
```

Reply:
ok, I'll change it after your PR is merged

Reply:
Thanks, I'll modify it